specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json

```diff
@@ -0,0 +1,1663 @@
+      "enum": [
+       "clientAuthentication",
+       "serverAuthentication"
+      ],
+      "x-ms-enum": {
+       "name": "CertificatePurpose",
+       "modelAsString": "True"
+      }
+     },
+     {
+      "name": "certificate.created",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "format": "date-time"
+     },
+     {
+      "name": "certificate.lastUpdated",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "format": "date-time"
+     },
+     {
+      "name": "certificate.hasPrivateKey",
+      "in": "query",
+      "required": false,
+      "type": "boolean"
+     },
+     {
+      "name": "certificate.nonce",
+      "in": "query",
+      "required": false,
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/api-version"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "OK",
+      "schema": {
+       "$ref": "#/definitions/VerificationCodeResponse"
+      }
+     },
+     "default": {
+      "description": "DefaultErrorResponse",
+      "schema": {
+       "$ref": "#/definitions/ErrorDetails"
+      }
+     }
+    }
+   }
+  },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/certificates/{certificateName}/verify": {
+   "post": {
+    "tags": [
+     "POST"
+    ],
+    "operationId": "DpsCertificate_VerifyCertificate",
+    "x-ms-examples": {
+        "DPSVerifyCertificate": { "$ref": "./examples/DPSVerifyCertificate.json" }
+    },
+    "consumes": [
+     "application/json"
+    ],
+    "produces": [
+     "application/json"
+    ],
+    "parameters": [
+     {
+      "name": "certificateName",
+      "in": "path",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "name": "request",
+      "in": "body",
+      "required": true,
+      "schema": {
+       "$ref": "#/definitions/VerificationCodeRequest"
+      }
+     },
+     {
+      "$ref": "#/parameters/subscriptionId"
+     },
+     {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "name": "provisioningServiceName",
+      "in": "path",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "name": "certificate.name",
+      "in": "query",
+      "required": false,
+      "type": "string"
+     },
+     {
+      "name": "certificate.rawBytes",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "format": "byte"
+     },
+     {
+      "name": "certificate.isVerified",
+      "in": "query",
+      "required": false,
+      "type": "boolean"
+     },
+     {
+      "name": "certificate.purpose",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "enum": [
+       "clientAuthentication",
+       "serverAuthentication"
+      ],
+      "x-ms-enum": {
+       "name": "CertificatePurpose",
```

Comment:
This can't have quotes.

Please use 
`"modelAsString": true`

Reply:
fixed